scripts/test-versions.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+set -ueo pipefail
+
+main_cfg_dir=$1
+command=$2
+
+bucket="bucket-123"
+test_files_dir="${main_cfg_dir}/testfiles"
+stage1_dst_dir="${main_cfg_dir}/stage1"
+stage2_dst_dir="${main_cfg_dir}/stage2"
+
+setup(){
+    mkdir -p "$test_files_dir" "$stage1_dst_dir" "$stage2_dst_dir"
+    random_bytes_file () {
+        size=$1
+        output=$2
+	    head -c $size </dev/urandom > $output
+    }
+    random_bytes_file "2048"   "$test_files_dir/small-upload-testfile"          # create 2kb file of random bytes (inline)
+    random_bytes_file "5242880" "$test_files_dir/big-upload-testfile"            # create 5mb file of random bytes (remote)
+    random_bytes_file "134217728" "$test_files_dir/multisegment-upload-testfile"   # create 128mb file of random bytes (remote)
+
+    echo "setup test successfully"
+}
+
+echo "Begin test-versions.sh, storj-sim config directory:" ${main_cfg_dir}
+
+echo "which storj-sim: $(which storj-sim)"
+# shasum $(which storj-sim)
```

Comment:
I see this line and a few below are commented-- were those used for debugging or were they supposed to be put back in?